### ISSUE TITLE
question: additional tag when publish

### ISSUE THREAD

**Issue by tuananh** [chainguard-dev/apko#451]
does apko publish suppports it or it's out of the scope of the tool? maybe in CI with use of tool like crane?

**Comment by coreydaley-cg** [comment]
Yes, `apko publish config.yaml tag1 tag2 ...` applies every tag you pass. Closing as answered.
